build-system/tasks/jsify-css.js

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
i can split it into a separate PR. thought i would need it but browserify didnt blow up for some reason w/o the CSS dependency being there. will investigate